tests/e2e/core-tests/test-data/sample_products_override.csv

```diff
@@ -0,0 +1,26 @@
+﻿ID,Type,SKU,Name,Published,"Is featured?","Visibility in catalog","Short description",Description,"Date sale price starts","Date sale price ends","Tax status","Tax class","In stock?",Stock,"Backorders allowed?","Sold individually?","Weight (lbs)","Length (in)","Width (in)","Height (in)","Allow customer reviews?","Purchase note","Sale price","Regular price",Categories,Tags,"Shipping class",Images,"Download limit","Download expiry days",Parent,"Grouped products",Upsells,Cross-sells,"External URL","Button text",Position,"Attribute 1 name","Attribute 1 value(s)","Attribute 1 visible","Attribute 1 global","Attribute 2 name","Attribute 2 value(s)","Attribute 2 visible","Attribute 2 global","Meta: _wpcom_is_markdown","Download 1 name","Download 1 URL","Download 2 name","Download 2 URL"
```

Comment:
The product image URLs (such as `https://woocommercecore.mystagingwebsite.com/wp-content/uploads/2017/12/logo-1.jpg` should be fairly stable, right? I believe the importer fails if the images can't be found, so just wanted to check to verify we expect this site to be around for a while (considering those images were uploaded in 2017 I assume so, but figured I'd double check).

Reply:
> The product image URLs (such as https://woocommercecore.mystagingwebsite.com/wp-content/uploads/2017/12/logo-1.jpg) should be fairly stable, right?

Those are the URLs that are distributed with WooCommerce: https://github.com/woocommerce/woocommerce/blob/trunk/sample-data/sample_products.csv

Reply:
Ah! Thank you for clarifying that @rrennick!